Code/Tools/AssetBundler/source/models/AssetBundlerAbstractFileTableModel.cpp

```diff
@@ -61,8 +61,12 @@ namespace AssetBundler
         {
             AZStd::string absolutePath = filePath.toUtf8().data();
             if (AZ::IO::FileIOBase::GetInstance()->Exists(absolutePath.c_str()))
-            { 
-                AZStd::string projectName = pathToProjectNameMap.at(absolutePath);
+            {
+                AZStd::string projectName;
+                if (pathToProjectNameMap.contains(absolutePath))
+                {
+                    projectName = pathToProjectNameMap.at(absolutePath);
+                }
```

Comment:
What is the project name in this instance?
Is the O3DE Project name?

If so, why is that needed. There is only one project that can be active in a given application.
At that point AZ::Utils::GetProjectName can just be called.

Reply:
It's not always the O3DE Project Name, it's the name of the thing that holds the object at that path: Gem name, scan folder, or game project. It's not a great name for that container, it could be better named.

For example, the key "C:\\git\\o3de\\Gems\\Atom\\Bootstrap\\Assets\\seedList.seed" resolves to "Atom_Bootstrap Gem", and the key "C:\\git\\o3de\\Assets\\Engine\\SeedAssetList.seed", resolves to "Assets\\Engine"

Reply:
Thanks for the clarification.
So it is really the key supplied for the ScanFolder?

Reply:
Mostly, yeah. It's a human readable display name for the scan folder, but that's not a concept for the scan folder directly, at least in the asset database, so I think the bundler generates that in other ways.